server/src/middlewares/webAppProxyMiddleware.ts

```diff
@@ -100,7 +100,7 @@ export async function metasResponseInterceptor(
   }
 
   try {
-    return await replaceHtmlMetas(html, endpoint, entityKey as EntityKeys, Number(id))
+    return await replaceHtmlMetas(html, endpoint || '', entityKey as EntityKeys, Number(id))
```

Comment:
Pour fix `Argument of type 'string | undefined' is not assignable to parameter of type 'string'`

Reply:
je propose de changer la condition au dessus en

```ts
  if (!endpoint || !id) {
    return html
  }
```